src/mhd2xds.coffee

```diff
@@ -1,8 +1,25 @@
 js2xml = require 'js2xmlparser'
-uuid = require 'node-uuid'
+uuid = require 'uuid'
+dom = require('xmldom').DOMParser
+xpath = require 'xpath'
 xds = require './xds'
 
-exports.mhd1Metadata2Xds = (metadata) ->
+exports.mhd1Metadata2XdsForMomconnect = (metadata, cda) ->
+  cda = cda.trim()
+  doc = new dom().parseFromString(cda)
+
+  select = xpath.useNamespaces
+    'v3': 'urn:hl7-org:v3'
+
+  birthDate = select 'string(//v3:ClinicalDocument/v3:recordTarget/v3:patientRole/v3:patient/v3:birthTime/@value)', doc
+  birthDate = birthDate.replace '%', ''
+
+  authorId = select 'string(//v3:ClinicalDocument/v3:author/v3:assignedAuthor/v3:id/@extension)', doc
+  if authorId.length is 0 then authorId = 'unknown'
+  authorAssigningAuthorityId = select 'string(//v3:ClinicalDocument/v3:author/v3:assignedAuthor/v3:id/@root)', doc
```

Comment:
Ok I'l give it a go, do you know how it is possible to tell which author element it is (ie. provider or software)?